docs/i18n/ru/docusaurus-plugin-content-docs/current/usage/call.md

```diff
@@ -11,18 +11,18 @@ pagination_label: Вызов сервиса
 
 Сервисы могут быть вызваны только через методы `.call` и `.call!`.
 
-Вызов через метод `.call` будет падать с ошибкой только в том случае, если он перехватит исключение в `input` аргументах.
-Ошибки, возникшие в атрибутах `internal` и `output`, а также ошибки, возникшие в методах — все это будет собрано в `Result` сервиса.
+Вызов через метод `.call` будет падать с ошибкой только в том случае, если он перехватит исключение в input-аргументах.
+Ошибки, возникшие в атрибутах internal и output, а также ошибки, возникшие в методах — все это будет собрано в `Result` сервиса.
```

Comment:
@nikogdanikomu Please translate this line:

```md
Ошибки, возникшие в атрибутах internal и output, а также ошибки, возникшие в методах — все это будет собрано в `Result` сервиса.
```

It is necessary to keep the reference to `Result` as a class.